File: internal/content/testdata/bench/go/find-celebrity-in-party.go
func find_celebrity(knows_matrix [][]bool) int {
	n := len(knows_matrix)
	candidate := 0
	for i := 1; i < n; i++ {
		if knows_matrix[candidate][i] {
			candidate = i
		}
	}
	for i := 0; i < n; i++ {
		if i != candidate {
			if knows_matrix[candidate][i] || !knows_matrix[i][candidate] {
				return -1
			}
		}
	}
	return candidate
}

File: internal/content/testdata/bench/csharp/find-celebrity-in-party.cs
public class Solution {
    public long find_celebrity(bool[][] knows_matrix) {
        int n = knows_matrix.Length;
        int candidate = 0;
        for (int i = 1; i < n; i++) {
            if (knows_matrix[candidate][i]) {
                candidate = i;
            }
        }
        for (int i = 0; i < n; i++) {
            if (i != candidate) {
                if (knows_matrix[candidate][i] || !knows_matrix[i][candidate]) {
                    return -1;
                }
            }
        }
        return candidate;
    }
}
